packages/guile-xyz/guile-json/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/aconchillo/guile-json
+TERMUX_PKG_DESCRIPTION="Guile-JSON supports parsing and building JSON documents according to the specification(http://json.org)."
+TERMUX_PKG_LICENSE="GPL3 or later"
+TERMUX_PKG_VERSION=3.1.0
+TERMUX_PKG_NAME=guile-json
```

Comment:
TERMUX_PKG_NAME should not be specified.
Package name are determined by package directory name.